### DISCUSSION TITLE
image docker recommend Microsoft OpenJDK

### DISCUSSION THREAD

**Discussion by kendyor** [microsoft/openjdk#142] [Q&A] [answered]
What image docker recommend for using Microsoft OpenJDK with AKS?

**Comment by karianna** [comment, marked as answer]
Any of the images at: https://hub.docker.com/_/microsoft-openjdk-jdk are recommended.

**Reply by kendyor** [reply]
Thanks, are you working seamlessly with AKS?

**Reply by brunoborges** [reply]
> Thanks, are you working seamlessly with AKS?

Mind to elaborate what does it mean to "work seamlessly with AKS"?

**Reply by kendyor** [reply]
Hello, of course, what I meant is if for my uploads in production I could use the images and in case of an error would I have the support of microsoft?
